python/lsst/ts/observatory/control/auxtel/atcalsys.py

```diff
@@ -226,19 +253,21 @@ async def run_calibration_sequence(
 
         calibration_type = getattr(CalibrationType, str(config_data["calib_type"]))
         if calibration_type == CalibrationType.WhiteLight:
-            calibration_wavelenghts = np.array([float(config_data["wavelength"])])
+            calibration_wavelengths = np.array([float(config_data["wavelength"])])
         else:
             wavelength = float(config_data["wavelength"])
             wavelength_width = float(config_data["wavelength_width"])
```

Comment:
Can we change `wavelength_width` to something like `filter_width` or `band_width`. I find this term confusing. And why not just have a `wavelength_start` and `wavelength_end` with resolution?

Reply:
I think this is a matter of personal preference. I find passing center wavelength + width more expressive. This is not a filter or band width, but rather the "scan width". I think we can certainly change these to something else if you think they will be more clear, but let's not do it as part of this ticket. Let's focus on what is being doing here instead. 

Feel free to open a ticket for updating this though, and we can discuss in the ticket what would be a better solution.